src/intrinsics/ecma262/Object.js

```diff
@@ -113,8 +115,12 @@ export default function(realm: Realm): NativeFunctionValue {
         // For now, just throw if this happens.
         let to_keys = to.$OwnPropertyKeys();
         if (to_keys.length !== 0) {
+          unsafe_to_be_simple = true;
+          to_must_be_simple = false;
```

Comment:
These won't matter since you're just going to throw before this function can do anything useful with them anyway.